libs/pilight/events/actions/pushsafer.c

```diff
@@ -0,0 +1,311 @@
+/*
+	Copyright (C) 2013 - 2014 CurlyMo
+	
+	Copyright (C) 2016 Pushsafer.com Kevin Siml > Appzer.de
```

Comment:
Can you make the Copyright one single line with your info (and mine if you like).